t/fuzzing/public.py

```diff
@@ -20,14 +20,15 @@
 import subprocess
 import os
 import psutil
+from boofuzz import *
```

Comment:
still not fixed

Reply:
This is the official usage. If it is changed to `import boofuzz`, the code will be very complicated. @moonming